tlf/id.go

```diff
@@ -88,6 +88,43 @@ func TypeFromFolderType(ft keybase1.FolderType) Type {
 	}
 }
 
+// KeyingType represents a TLF keying mode. It normally have the same values
+// as Type.
+type KeyingType Type
+
+const (
+	// UnknownKeying is a placeholder type for when TLF keying mode is unknown.
+	UnknownKeying = KeyingType(Unknown)
+	// PrivateKeying specifies the  TLF keying mode used in classic private TLF
```

Comment:
extra space in "the  TLF" (here and below).

Reply:
In the name of human's curiosity in exploring the space.